spec/app_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe App do
+  before do
+    @store = Store.new
+    @store.add('POST /hook', 'https://www.example.com')
+  end
+
+  after do
+    @store.delete_all
+  end
+
+  it do
+    env = Rack::MockRequest.env_for(
+      '/hook',
+      'REQUEST_METHOD' => 'POST',
+      'HTTP_USER_AGENT' => 'curl/7.43.0',
+      'HTTP_ACCEPT' => '*/*',
+      'CONTENT_TYPE' => 'application/x-www-form-urlencoded',
+      :input => 'foo=bar&quux=bla'
+    )
+
+    VCR.use_cassette("postman") do
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.